src/System.Text.Json/docs/writable_json_dom_spec.md

```diff
@@ -230,8 +254,12 @@ Mailbox.SendAllEmployeesData(employees.AsJsonElement());
 * Do we want to change `JsonNumber`'s backing field to something different than `string`?     
     Suggestions: 
     - `Span<byte>` or array of `Utf8String`/`Char8` (once they come online in the future) / `byte`  
-    - internal types that inherit from `JsonNumber`  
+    - Internal types that are specific to each numeric type in .NET with factories to create JsonNumber 
+    - Internal struct field which has all the supported numeric types
+    - Unsigned long field accompanying string to store types that are <= 8 bytes long
 * Do we want to support creating `JsonNumber` from `BigInterger` without changing it to string?
+* Should `ToString` on `JsonBoolean` and `JsonString` return the .NET or JSON representation?
+* Do we want to keep implicict cast operators (even though for `JsonNumber` it would mean throwing in some cases, which is against FDG)?
```

Comment:
typo:
```suggestion
* Do we want to keep implicit cast operators (even though for `JsonNumber` it would mean throwing in some cases, which is against FDG)?
```